packages/@glimmer/util/lib/platform-utils.ts

```diff
@@ -39,7 +39,7 @@ export type Lit = string | number | boolean | undefined | null | void | {};
 
 export const tuple = <T extends Lit[]>(...args: T) => args;
 
-export function enumerableSymbol(key: string): any {
+function enumerableSymbol(key: string): any {
```

Comment:
We should still probably export so that we can drop the duplicate copy over in Ember

Reply:
If that's the case I will also move `intern` upstream, cause I don't want us to accidentally de-opt without using that for enumerable symbols

Reply:
Ya, that makes sense to me.